npm/src/controller/connection/saml.ts

```diff
@@ -17,6 +17,18 @@ import {
 import saml20 from '@boxyhq/saml20';
```

Comment:
Please make changes to `@swagger` definitions. You can find it `/npm/src/controller/api.ts`